src/pages/tokens/[tokenid].js

```diff
@@ -433,7 +443,9 @@ export default function Token(props) {
         <TagList>
           <TreeTag
             key="created-at"
-            TreeTagValue={new Date(token.created_at).toLocaleDateString()}
+            TreeTagValue=<time dateTime={new Date(token.created_at)}>
```

Comment:
We can just pass `token.created_at` to the `dateTime` prop

```suggestion
            TreeTagValue=<time dateTime={token.created_at}>
```